ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -71,6 +71,7 @@
   public final Counter blobDeletedErrorCount;
   public final Counter blobDoesNotExistErrorCount;
   public final Counter blobExpiredErrorCount;
+  public final Counter rangeNotSatisfiableErrorCount;
```

Comment:
as commented earlier, I would prefer to have separate metrics for GetBlob with range requests.